translations/strings.yaml

```diff
@@ -76,6 +76,9 @@ serversView:
   searchPlaceholder: Search countries
   searchNoResultsLabel: No results. Try a different search
 
+languageView:
+  searchPlaceholder: Search languages
```

Comment:
I know it might seem obvious, but English is confusing. Could we add a comment that this is for a field searching **through** languages?

Reply:
Sure — makes sense. Added a comment to the string.